src/Components/ServiceScene/LogsTableScene.tsx

```diff
@@ -117,16 +117,5 @@ const getStyles = (theme: GrafanaTheme2) => ({
     width: '100%',
     height: '100%',
     label: 'panel-wrapper-table',
-
```

Comment:
There's still 1 more of these in the PanelMenu.tsx.
Also I think we'll need to remove all of the comments and add `.setShowMenuAlways(true)` to all of the panels
``` 
// .setShowMenuAlways(true)
```

Search for 11.5 in the code and they should all come up.

Reply:
aaaand another one in PatternsFrameScene.tsx: 
```
    // Hide header on hover hack
    '.show-on-hover': {
      display: 'none',
    },
```

Reply:
Oh wow, I missed so many ty! Updated!
Should I also delete these functions in `TagKeysProviders.ts`
```
// @todo delete after min supported grafana is upgraded to >=11.6
  // see ced526b3e37baded9082ffc3c2378a21201801b6 before this all got messed up
  const fetchDetectedFieldsFn =
    (datasource &&
      typeof languageProvider.fetchDetectedFields === 'function' &&
      languageProvider.fetchDetectedFields.bind(languageProvider)) ||
    function (opts: DetectedFieldQueryOptions) {
      return fetchDetectedFields(datasource, opts);
    };
```

Reply:
lol nm, found the issue you already created for this.